### ISSUE TITLE
Bug with recent room messages

### ISSUE THREAD

**Issue by bjoernw** [Viascom/hipchat-api#27]
I'm trying to cut down the message collection by "not-before" some ID and a max-results of 15. Somehow I always get the default 75 items and the "not-before" is ignored as well. 
```
ViewRecentRoomHistory history = new ViewRecentRoomHistory();
history.setMax_results(15);
history.setNot_before(latestRoomIds.get(room));
ViewRecentRoomHistoryResponse rep = hipChat.roomsApi().viewRecentRoomHistory(room, history);
```

When I debugged your code it looks like the final request is this:
```
https://api.hipchat.com/v2/room/MyRoom/history/latest?timezone=UTC&not_before=5110cadc-056f-4e92-860a-ff8f75cf905f&max_results=15&include_deleted=true
```

However, the documentation says that those parameters should have dashes and not underscores. 
"not-before" vs. "not_before"
Docs: https://www.hipchat.com/docs/apiv2/method/view_recent_room_history

Maybe something is ignoring your annotations?

Cheers,
Bjoern

**Comment by nik-sta** [comment]
Hi @bjoernw :-)  ... 

A huge thanks for your bug report. Your analysis was perfect. The annotation was ignored. This bug is now fixed in the version: 2.1-RC3. This bug influenced also other requests which are now fixed as well.

You should be able to use this new release candidate in some minutes (approximately in 20 minutes). We are uploading it now to the maven central repository,

Cheers, Nikola

**Comment by bjoerndanz** [comment]
Thanks for the quick fix and all your work on this library!
